src/modules/edc-demo/pages/catalog-browser/asset-details/asset-details.component.html

```diff
@@ -70,27 +70,33 @@ <h2>Policy Details</h2>
   </div>
 
   <mat-list dense class="catalogue-meta-data-list" fxLayout="row wrap">
-    <mat-list-item>
-      <mat-icon mat-list-icon>policy</mat-icon>
-      <div mat-line class="asset-property-name">Policy ID</div>
-      <div mat-line>{{contractOffer.id}}</div>
-    </mat-list-item>
-    <mat-list-item>
-      <mat-icon mat-list-icon>ads_click</mat-icon>
-      <div class="asset-property-name" mat-line>Target</div>
-      <div mat-line>{{contractOffer.policy.target ?? '- -'}}</div>
-    </mat-list-item>
     <mat-list-item>
       <mat-icon mat-list-icon>handshake</mat-icon>
-      <div class="asset-property-name" mat-line>Verpflichtungen</div>
-      <div mat-line *ngFor="let obligation of contractOffer.policy.obligations!">{{obligation.action?.type}} {{obligation.target}}</div>
+      <div class="asset-property-name" mat-line>Obligations</div>
```

Comment:
Warum hier auf einmal Englisch?

Reply:
Fand es irgendwie schöner hier die gängigen englischen Bezeichnungen einzuführen und es ist sowieso immer ein ziemlicher Denglisch-Misch-Masch. Also dann wäre man zumindest bei Permissions, Obligations und Prohibitions konsistent mit dem EDC-Sprech weil die Begriffe auch so in der Policy auftauchen. Wobei du hattest es auf der Plattform glaube ich auf Deutsch geschrieben, also ja, man müsste sich auf eins von beiden einigen.